build-support/bin/generate_github_workflows.py

```diff
@@ -67,6 +67,20 @@ def checkout() -> Sequence[Step]:
     ]
 
 
+def setup_toolchain_auth() -> Sequence[Step]:
+    return [
+        {
+            "name": "Setup toolchain auth",
+            "if": "github.event_name == 'push'",
+            "run": dedent(
+                """\
+                echo TOOLCHAIN_AUTH_TOKEN="{{ secrets.TOOLCHAIN_AUTH_TOKEN }}" >> $GITHUB_ENV
```

Comment:
Should this be?:
```suggestion
                echo TOOLCHAIN_AUTH_TOKEN="${{ secrets.TOOLCHAIN_AUTH_TOKEN }}" >> $GITHUB_ENV
```
https://docs.github.com/en/actions/reference/encrypted-secrets

Reply:
🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼 🤦🏼